### ISSUE TITLE
Bug in `Run commands from iPXE shell` functionality

### ISSUE THREAD

**Issue by TomaszAIR** [Dasharo/dasharo-issues#387]
**DTS version**

v1.1.1 and v1.1.2-dev

**Affected component(s) or functionality**

Run commands from iPXE shell

[1] https://docs.dasharo.com/dasharo-tools-suite/documentation/#run-commands-from-ipxe-shell

**Brief summary**

Executing steps from documentation ends on `iPXE error (some_error_ID)

**How reproducible**

Every time

**How to reproduce**

Described in linked documentation

**Expected behavior**

ipxe-commands script populated in DTS rootfs and executed on boot

**Actual behavior**

iPXE cannot start, if we remove URL from first `initrd` entry (as mentioned in documentation), the DTS boots but script is not populated into path `/sbin/ipxe-commands` on DTS rootfs. 

**Screenshots**

**Additional context**

We believe that when functionality was introduced we used different version of iPXE, maybe something changed there. We did not notice which version was used when testing.

**Solutions you've tried**

**Comment by TomaszAIR** [comment]
I tested this again using DTS v1.2.18. I took bzImage and cpio from http://boot.dasharo.com/dts/v1.2.18/ placed it in one dir, added there ipxe file

```
#!ipxe
#
kernel http://<IP_ADD>/bzImage-v1.2.18 root=/dev/nfs initrd=http://<IP_ADD>/dts-base-image-v1.2.18.cpio.gz
initrd http://<IP_ADD>/dts-base-image-v1.2.18.cpio.gz
module http://<IP_ADD>/custom-script /sbin/ipxe-commands mode=755
boot
```
simple script to create file under homedir

```
#!/bin/bash

echo "Test ipxe commands script" > ~/file
```

and started http server from this dir. Then booted DTS several times, did not get any errors. Script was executed and file created.

```
root@DasharoToolsSuite:~# cat file 
Test ipxe commands script
``` 

I think that we can add a script to meta-dts that would automate using this functionality (e.g. download image from boot.dasharo.com and start http server) and then we could point to it in the documentation.

**Comment by WiktorG351** [comment]
Hi, this has been adressed in [this pr](https://github.com/Dasharo/meta-dts/pull/96) to `meta-dts`. The documentation has also been [updated](https://github.com/Dasharo/docs/pull/764) accordingly.

**Comment by TomaszAIR** [comment]
Fixes merged in v1.2.19, documentation is also updated https://docs.dasharo.com/dasharo-tools-suite/documentation/#run-commands-from-ipxe-shell-automatically.

@artur-rs I think we can close this issue.
